website/client/src/components/inventory/stable/petItem.vue

```diff
@@ -168,34 +174,40 @@ export default {
     isAllowedToFeed () {
       return isAllowedToFeed(this.item, this.userItems);
     },
-    getPetItemClass () {
+    itemClass () {
+      if (this.isOwned() || (this.mountOwned() && this.isHatchable()) || this.isSpecial()) {
```

Comment:
missing the "isHatchable" alone  - see https://github.com/HabitRPG/habitica/pull/15242/files#diff-4b9c11aea9816b0f238d1da7970e671a1189835e1b0bfe7aa061b48bd3401262L190  so I guess this `&&` can be changed to a `||` and then it should be fine :)